Mondee Watch App/View/Guide/BeforeGameCountView.swift

```diff
@@ -8,61 +8,84 @@
-                    .fontWeight(.black)
-                    .fontDesign(.rounded)
-                    .foregroundStyle(
-                        Color.black
-                        .shadow(.inner(color: .pink.opacity(0.5), radius: 4, x: 2, y: 2))
-                        .shadow(.inner(color: .blue, radius: 4, x: -2, y: 0))
-                        .shadow(.inner(color: .yellow.opacity(0.5), radius: 4, x: 1.5, y: -2))
-                        .shadow(.inner(color: .white.opacity(0.5), radius: 4, x: 0, y: 2))
-                    )
-                    .padding()
+        if countdown > 0 {
+            VStack {
+                Circle()
+                    .trim(from: 0, to: progress)
+                    .stroke(style: StrokeStyle(
+                        lineWidth: 10,
+                        lineCap: .round
+                    ))
+                    .modifier(BubbleFontModifier())
+                    .frame(width: 153, height: 153)
+                    .rotationEffect(Angle(degrees: -90))
+                    .animation(.easeInOut, value: progress)
                     .onAppear {
                         startCountdown()
                     }
-            } else {
-                Text("GO !")
-                    .font(.system(size: 80))
-                    .fontWeight(.black)
-                    .fontDesign(.rounded)
-                    .foregroundStyle(
-                        Color.black
-                        .shadow(.inner(color: .pink.opacity(0.5), radius: 4, x: 2, y: 2))
-                        .shadow(.inner(color: .blue, radius: 4, x: -2, y: 0))
-                        .shadow(.inner(color: .yellow.opacity(0.5), radius: 4, x: 1.5, y: -2))
-                        .shadow(.inner(color: .white.opacity(0.5), radius: 4, x: 0, y: 2))
+                    .background(
+                        Circle()
+                            .stroke(style: StrokeStyle(
+                                lineWidth: 10,
+                                lineCap: .round
+                            ))
+                            .foregroundColor(.gray.opacity(0.2))
                     )
-                    .padding()
-                    .onAppear{
-                        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
-                            gameStatus = .play
-                        }
+                    .overlay(
+                        countShow
+                        ?
+                            Text("\(countdown)")
+                                .font(.system(size: 100))
+                                .modifier(BubbleFontModifier())
+                        :
+                            Text("Ready")
+                                .font(.system(size: 30))
+                                .modifier(BubbleFontModifier())
+   
+                    )
+            }
+//            .onAppear{
+//                progress -= 1 / CGFloat(countdownDuration)
+//            }
```

Comment:
```suggestion
//            .onAppear{
//                progress -= 1 / CGFloat(countdownDuration)
//            }
```
안쓰는 코드는 제거해주시면 감사해요 😊